module/button.py

```diff
@@ -1,18 +1,61 @@
 from telegram import Update
 from telegram.ext import CallbackContext
+from module.add_book import add_book
+from module.add_item import add_item
 from module.create_connection import create_connection
-from module.shared import DB_PATH
+from module.shared import DB_PATH, error_message
+from module.add_book import add_book
+from module.add_item import add_item
+from module.manage_requests import delete_request, update_similar_requests
 
 
 def button(update: Update, context: CallbackContext) -> None:
     chat_id = update.effective_chat.id
     query = update.callback_query
-    conn = create_connection(DB_PATH)
-    if conn:
+    
+    if query.data.split(';')[0] == 'delete':
```

Comment:
you have two identical if, they should be `if` and `elif`, moreover avoid to do the same big code `query.data.split(';')[0]`. save it into a variable or use `match` (it's `switch-case` in other languages)

Moreover, you could avoid magic strings like `delete` or `new_request`, you could save them into constant variables.

Reply:
Like the comment above, `query.data.split(';')[0]` will have only two values: `delete ` or `new_request`. Only one of those ifs will compute.
Good idea for the constants, I'll add those to the `shared.py` module

Reply:
I don't know if I am missing something but there is no return in the first if, so the second if will be triggered anyway

So the second if will trigger again a split in `query.data.split(';')[0]` and a comparison with `'delete'`, with the elif you'll skip it, but anyway would be better to wrap query.data.split into a separate variable or into `match` and use constants

Reply:
```python
if something == 'asd':
   return
  
if something == 'lol':
  ...
```

that's ok

```python
if something == 'asd':
   ...
   
if something == 'lol':
  ...
```

this requires `elif`, but still, it's better `match` (switch case)